variables.tf

```diff
@@ -30,8 +30,8 @@ variable "enable_otel_collector" {
 }
 
 variable "enable_prometheus_operator" {
-  default     = true
-  description = "Enables the Prometheus Operator and other components via kube-stack-prometheus. Set to \"true\" by default."
+  default     = false
```

Comment:
This will disable the whole kube-prometheus-stack helm chart.

We can set `prometheusOperator.enabled=false` if we wanna just disable the prometheus operator.